x-pack/platform/plugins/private/discover_enhanced/ui_tests/parallel_tests/value_suggestions.spec.ts

```diff
@@ -22,20 +22,16 @@ spaceTest.describe(
       });
     });
 
-    spaceTest.afterAll(async ({ scoutSpace }) => {
-      await scoutSpace.uiSettings.unset(
-        'doc_table:legacy',
```

Comment:
I see that this `doc_table` setting has not been taken over (doesn't exist on `main`). Was that an intentional change?

Reply:
good catch, I was trying to avoid a change here.